spec/models/work_spec.rb

```diff
@@ -162,29 +189,11 @@
     end
   end
 
-  context "when updating the ARK" do
-    before { allow(Ark).to receive(:update) }
-    let(:ezid) { "ark:/99999/dsp01qb98mj541" }
-
-    around do |example|
-      Rails.configuration.update_ark_url = true
-      example.run
-      Rails.configuration.update_ark_url = false
-    end
-
-    it "updates the ARK metadata" do
```

Comment:
I would expect this test would still be valid.  You would just have to approve the item in the test.

Reply:
It is still valid. The way Git aligns the difference looks like I deleted the test but in fact I just moved it to a different section (see lines 123-129) and added another test related to ARK (lines 131-137)